ui/src/app/records/custom-editor/circulation-settings/circulation-policy/circulation-policy.component.html

```diff
@@ -26,8 +26,14 @@ <h1 translate>Circulation Policy</h1>
     <div class="form-group row">
       <label for="name" class="col-sm-2 col-form-label required" translate>Name</label>
       <div class="col-sm-10">
-        <input class="form-control" id="name" formControlName="name"
-          placeholder="{{ 'Please insert a name' | translate }}" required>
+        <input  
```

Comment:
I will do the review for all ui file in an other PR to the `rero-ils-ui` project.